application/src/js/cms/rd-table-objects.js

```diff
@@ -114,7 +114,7 @@ function simpleTable(data, title, subtitle, footer, category_column, parent_colu
         tableData = adjustSimpleTableDataForParents(tableData);
     }
 
-    var index_column = index_column_caption == null ? category_column : index_column_caption;
+    var index_column = !index_column_caption ? category_column : index_column_caption;
```

Comment:
super minor point, but would this be slightly less confusing as the reverse, ie:

```javascript
var index_column = index_column_caption ? index_column_caption : category_column;
```